src/Schema/Keywords/Type.php

```diff
@@ -38,56 +39,44 @@ class Type extends BaseKeyword
     public function validate($data, string $type, ?string $format = null) : void
     {
         try {
-            Validator::in([
-                0 => 'boolean',
-                1 => 'object',
-                2 => 'array',
-                3 => 'number',
-                4 => 'integer',
-                5 => 'string',
-                // Note that there is no null type; instead, the nullable attribute is used as a modifier of the base type.
-            ])->stringType()->assert($type);
-
             if ($this->parentSchema->nullable && $data === null) {
```

Comment:
should we skip this also? we've fixed this recently, didn't we? in another place

Reply:
Indeed, we did solve it here: `src/Schema/Validator.php` on lines 68-73. 
Looks like this is safe to remove.

Reply:
Removed